staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -55,8 +56,12 @@ func newETCD3Storage(c storagebackend.Config) (storage.Interface, DestroyFunc, e
 		cancel()
 		client.Close()
 	}
+	transformer := c.Transformer
+	if transformer == nil {
+		transformer = value.IdentityTransformer
```

Comment:
Are there plans to ever make something other than `IdentityTransformer` the out-of-the-box default?